wallet/how-to/use-non-evm-networks/index.md

```diff
@@ -0,0 +1,23 @@
+---
+description: Connect ...
+---
+
+# Connect non-EVM wallets to MetaMask using Snaps
+
+## Non-EVM Wallets
+
+Non-EVM wallets are cryptocurrency wallets that operate on blockchain networks not compatible with the Ethereum Virtual Machine (EVM). 
+Non-EVM wallets can't directly interact with MetaMask or other Ethereum-based applications. 
```

Comment:
Sentence maybe revisited based on the other review commens, emphasising that yes

 `Non-EVM wallets can't interact with metamask` 
 
 but also that 
 
 `metamask can't interact with dapps from non-EVM networks`.